azure-pipelines.yml

```diff
@@ -48,7 +49,10 @@ steps:
       python -m pip install --upgrade pip
       pip install wheel pep517 setuptools_scm
       node -v
-      pip install -i https://pypi.chia.net/simple/ miniupnpc==2.2.2
+      pip install -i https://pypi.chia.net/simple/ miniupnpc==2.1
```

Comment:
Careful - this is a regression from my changes over the weekend to miniupnpc to 2.2.2